src/lib/krb5/krb/mk_req_ext.c

```diff
@@ -253,6 +253,35 @@ krb5_mk_req_extended(krb5_context context, krb5_auth_context *auth_context,
     return retval;
 }
 
+static krb5_error_code
+add_ad_ap_options(krb5_context context, krb5_authdata **out)
+{
+    krb5_error_code code;
+    krb5_authdata ad, *list[2], **ifrel;
+    uint32_t ad_ap_options;
```

Comment:
This should be `uint8_t ad_ap_options[4];'.

Reply:
Given store_32_le is used on this variable, wouldn't it potentially cause alignment problems on some platform if it were declared as an array of bytes ?

Reply:
store_32_le() writes an integer value (the first argument) into four bytes (the second argument).  The pointer to the bytes is not required to be aligned.

Reply:
ah, good to know, I thought it might be using optimized instructions that do the swap, those tend to require alignment, so nevermind